client/src/index.css

```diff
@@ -76,3 +76,9 @@ body > .content,
 #react-root > :only-child {
   flex-grow: 1;
 }
+
+/* FP-546: If this style has become unnecessary, then remove it */
```

Comment:
**Note to Reviewers**: [FP-546](https://jira.tacc.utexas.edu/browse/FP-546) is a button component ticket that includes buttons that look like links. Design may opt for a button or link _appearance_, but our decision to use a `<button>` or `<a>` tag should be based on [button v. link accessibility](https://confluence.tacc.utexas.edu/display/~wbomar/Shared+UI+-+Accessibility+-+Button+vs.+Link).